sonic-net/sonic-utilities
# [spm]: Clean up timers auto generation logic

Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>



Config Reload Enhancements PR https://github.com/sonic-net/SONiC/pull/1203 does not completely remove TIMERs from SONiC Package Manager infra. This PR is intended to complete the original changes.

`Systemd` TIMERs infra was replaced by `hostcfgd` service management.
That was done to improve reliability of service management.

#### What I did
* Removed redundant TIMERs infra

#### How I did it
* Updated SPM auto generation logic

#### How to verify it
1. Install application extension
```bash
spm install --from-tarball <app_ext_path>
```
2. Make sure `delayed` flag is set
```bash
docker image inspect <app_ext_image> | jq '.[].Config.Labels["com.azure.sonic.manifest"]' | python -c 'import sys,ast; print(ast.literal_eval(sys.stdin.read()))' | jq .service.delayed
true
```
3. Check no TIMERs were generated

#### Previous command output (if the output of a command-line utility has changed)
* N/A

#### New command output (if the output of a command-line utility has changed)
* N/A

#### A picture of a cute animal (not mandatory but encouraged)
```
      .---.        .-----------
     /     \  __  /    ------
    / /     \(  )/    -----
   //////   ' \/ `   ---
  //// / // :    : ---
 // /   /  /`    '--
//          //..\\
       ====UU====UU====
           '//||\\`
             ''``
```
